src/api/resources/SSX/SSXDataCollectionProcessingCells.ts

```diff
@@ -1,9 +1,9 @@
-import { createAuthenticatedResource } from '../Base/Authenticated';
+import { AuthenticatedEndpoint } from 'api/resources/Base/Authenticated';
 import { SSXDataCollectionProcessingCellsSingletonBase } from 'models/SSXDataCollectionProcessingCells';
 
-export const SSXDataCollectionProcessingCellsResource =
-  createAuthenticatedResource({
-    path: '/ssx/datacollection/processing/cells/:dummy',
+export const SSXDataCollectionProcessingCellsEndpoint =
```

Comment:
I dont see a reference to this one in the views, so maybe unused in main?
now `SSXDataCollectionProcessingCellsEndpoint` rather than `SSXDataCollectionProcessingCellsResource`

Reply:
Yes I don't use it anymore, could be removed!

Reply:
removed